src/content/en/updates/2016/04/credential-management-api.md

```diff
@@ -365,9 +365,12 @@ You may override `navigator.credentials` and hook it to your Chrome Extension to
 `get()` or  `store()` credentials.
 
 ## Resources
+To learn more about the Credential Management API, checkout [Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/).
+
 * [API Specification](https://www.w3.org/TR/credential-management/)
 * [Spec discussions & feedback](https://github.com/w3c/webappsec-credential-management)
 * [MDN API Reference](https://developer.mozilla.org/en-US/docs/Web/API/Credential_Management_API)
+* [Credential Management API Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/)
```

Comment:
For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`